README.md

```diff
@@ -84,6 +84,18 @@ See the [storage capacity section](#capacity-support) below for details.
 
 * `--capacity-for-immediate-binding <bool>`: Enables producing capacity information for storage classes with immediate binding. Not needed for the Kubernetes scheduler, maybe useful for other consumers or for debugging. Defaults to `false`.
 
+##### Distributed provisioning
+
+* `--node-deployment`: Enables deploying the external-provisioner together with a CSI driver on nodes to manage node-local volumes. Off by default.
```

Comment:
I'm wondering if we can avoid flag skew issues like `--node-deployment=true` and `--capacity-controller-deployment-mode=central`.

Can change the capacity flag to be a boolean, and then determine which mode to use based on node deployment?

Reply:
> Can change the capacity flag to be a boolean, and then determine which mode to use based on node deployment?

In retrospect that would be nicer, but it is a bit late to change the semantic of the `--capacity-controller-deployment-mode` that way - we would have to do a major update of external-provisioner for that because it is a breaking change.

Reply:
It's an alpha feature, I think we can make breaking changes to it.

Reply:
So `--enable-capacity=true/false` instead of `--capacity-controller-deployment-mode=central` and the actual mode of operation (central vs. local) determined by `--node-deployment`?

Works for me, I just won't get to it today.

Reply:
That sounds good to me! Do you think enable-capacity will always be a permanent flag, or would it fit a feature gate better?

Reply:
I think a permanent flag is better. The feature causes additional work and might not be that relevant for some drivers (like those where all volumes are available everywhere), so being able to keep it turned off will remain useful.